fabfile/dev.py

```diff
@@ -457,13 +571,138 @@ def run_test_big_object():
         port = 8000+i
         alba.osd_stop(port)
         long_id = "%i_%i_%s" % (port, 2000, alba.local_nodeid_prefix)
-        local(" ".join([
+        cmd = [
             env['alba_bin'],
             'decommission-osd', '--long-id', long_id,
             '--config', arakoon_config_file
-        ]))
+        ]
+        if is_true(tls):
+            alba._extend_alba_tls(cmd)
+        local(" ".join(cmd))
 
     # TODO
     # wait for maintenance process to repair it
     # (give maintenance process a kick?)
     # report mem usage of proxy & maintenance process
+
+@task
+def run_tests_compat(xml = True):
+    def test(old_proxy, old_plugins, old_asd):
+        try:
+            alba.smoke_test()
+            def alba_cli(extra):
+                cmd = [env['alba_bin']]
+                cmd.extend(extra)
+                cmd_line = ' '.join(cmd)
+                local(cmd_line)
+
+            obj_name = 'alba_binary'
+            ns = 'demo'
+            alba_cli(['proxy-upload-object', ns , env['alba_bin'], obj_name])
+            alba_cli(['proxy-download-object', ns, obj_name, '/tmp/downloaded.bin'])
+            alba_cli(['delete-object', ns, obj_name,
+                      '--config', './cfg/albamgr_example_arakoon_cfg.ini'])
+
+
+        except Exception, e:
+            print e
+            with warn_only():
+                local ("which alba.0.6")
+                local ("pgrep -a alba")
+                local ("pgrep -a arakoon")
+                local ("which fuser")
+                local ("sudo fuser -n tcp 10000 8001")
+
+            raise
+
+    def deploy_and_test(old_proxy, old_plugins, old_asds):
```

Comment:
these arguments (old_proxy, old_plugins, old_asds) are unused, so you're probably not testing what you intended to test...

(I went looking for a bug in run_tests_compat as I still think some operations are not backwards compatible, see my other comment https://github.com/openvstorage/alba/pull/67#discussion_r45275986 )